lib/src/client/impl/exchange_impl.dart

```diff
@@ -63,4 +63,24 @@ class _ExchangeImpl implements Exchange {
     }
     return queue.consume(consumerTag: consumerTag, noAck: noAck);
   }
+
+  Future<Consumer> bindQueueConsumer(String queueName, List<String> routingKeys,
```

Comment:
Happy to merge this functionality but you first need to:

- update the relevant interface in `/lib/src/client/exchange.dart` to include the new method.
- add a unit-test for the new method in `test/lib/exchange_test.dart`.